TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/tests/test_cli.py

```diff
@@ -38,6 +39,15 @@ def env():
     return Environment(config_file=VALID_SERVICES_YAML)
 
 
+@pytest.fixture(autouse=True)
+def block_k8s_config_store_usage(monkeypatch):
```

Comment:
Thanks for finding this one!
There's also a smell that we should probably be using more injection so that these things don't happen in the first place.